drivers/adc/adc_mcux_adc16.c

```diff
@@ -62,6 +62,7 @@ struct mcux_adc16_data {
 #ifdef CONFIG_ADC_MCUX_ADC16_ENABLE_EDMA
 	const struct device *dev_dma;
 	struct adc_edma_config adc_dma_config;
+	bool batch_mode;	/* batch conversion mode */
```

Comment:
Where is `batch_mode` set? And what's the difference between this and `continuous_convert`?

Reply:
@heinwessels , this is leaved for your RFC get merge, and I will change accordingly. the continuous_convert in NXP hardware means it will continues convert sample so the adc data register will keep updated, until adc abort this. which is a pure hardware behavour. this is why I want to seperate this from your RFC's `continues`

Reply:
Ah, I understand. It's not 100% certain if it will get added to the API though, and even if will still be called `batch_mode`. But  I guess you can always change it in that case.